tests/api/route-services/projects.test.js

```diff
@@ -129,16 +128,14 @@ describe('tests for the projects service', () => {
 
     projectService.getProjectsFromIds = jest.fn().mockImplementation((x) => x);
 
-    projectService.getProjects(user)
-      .then((res) => {
-        expect(res).toEqual(expectedResult);
+    const res = await projectService.getProjects(user);
 
-        expect(projectService.getProjectsFromIds).toHaveBeenCalledWith(expectedResult);
-      })
-      .then(() => done());
+    expect(res).toEqual(expectedResult);
+
+    expect(projectService.getProjectsFromIds).toHaveBeenCalledWith(expectedResult);
   });
 
-  it('GetProjects gets all the projects across many pages of scan results when the first page is empty', async (done) => {
+  test('GetProjects gets all the projects across many pages of scan results when the first page is empty', async () => {
```

Comment:
why do you change to `test`? some cases are sill using `it`

Reply:
"it gets all the projects"
"test getProjects gets all the projects"